sound/soc/sof/ipc.c

```diff
@@ -206,6 +208,15 @@ static int tx_wait_done(struct snd_sof_ipc *ipc, struct snd_sof_ipc_msg *msg,
 	if (ret == 0) {
 		dev_err(sdev->dev, "error: ipc timed out for 0x%x size %d\n",
 			hdr->cmd, hdr->size);
+
+		ipc_rec = snd_sof_dsp_read(sdev, HDA_DSP_BAR,
+					   HDA_DSP_SRAM_REG_FW_IPC_RECEIVED_COUNT);
+		ipc_proc = snd_sof_dsp_read(sdev, HDA_DSP_BAR,
+					    HDA_DSP_SRAM_REG_FW_IPC_PROCESSED_COUNT);
+
+		dev_dbg(sdev->dev, "ipc received: %u processed: %u\n",
+			ipc_rec, ipc_proc);
+
```

Comment:
this should be dev_err()
Also it's only supported for HDaudio so the right place to add this log is in intel/hda.c (hda_ipc_dump)

Reply:
Ok, you wrote that you want it on IPC timeout so I wanted to be it closer to ipc timeout message, but yea, it's not worth polluting ipc with hda for that, so i'll move it to hda.

Reply:
@plbossart Moved it to `hda_ipc_dump`